### ISSUE TITLE
New change to Bristleback Aghs doesnt work with Octarine Core cast range increase

### ISSUE THREAD

**Issue by vkarthikeya04s** [ValveSoftware/Dota2-Gameplay#2374]
### Ability name

Viscous Nasal Goo

### Description

New Aghs makes goo affect all units with current cast range of goo. But it only affects the basic range..and does not take into account of Octarine core or Aether lens extra range.

### Example Match ID (and possibly Timestamp)

1234

**Comment by Digirolamo** [comment]
Can confirm as of 4:40pm 8/25/2022 this is still the case:

Without Talent, left with aether and no aghs, right with aether and aghs
![image](https://user-images.githubusercontent.com/3914192/186764508-7f85debd-f641-420f-9a7c-d3f28bcc44bf.png)

**Comment by Philosoph228** [comment]
Because the spell becomes AOE with Aghanim. The lens/octarine enhance cast range only for _targeted_ spells. It's intended.

**Comment by shaileshgarg** [comment]
> Because the spell becomes AOE with Aghanim. The lens/octarine enhances cast range only for _targeted_ spells and items. It's intended.

Yes after Specter it is a No Target Spell, and not a Unit Target Spell. Other Similar Spells like Husker Inner Fire,, CM Ulti, Troll Whirling Axe Melee, also don't change range with Octarine Core.

**Comment by Digirolamo** [comment]
> Because the spell becomes AOE with Aghanim. The lens/octarine enhances cast range only for _targeted_ spells and items. It's intended.

> Yes after Specter it is a No Target Spell, and not a Unit Target Spell. Other Similar Spells like Husker Inner Fire,, CM Ulti, Troll Whirling Axe Melee, also don't change range with Octarine Core.

The 7.32 patch changes Bristle's AOE to "Scepter radius changed from 800 to current Cast Range of Viscous Nasal Goo"

**Comment by shaileshgarg** [comment]
> > Because the spell becomes AOE with Aghanim. The lens/octarine enhances cast range only for _targeted_ spells and items. It's intended.
> 
> > Yes after Specter it is a No Target Spell, and not a Unit Target Spell. Other Similar Spells like Husker Inner Fire,, CM Ulti, Troll Whirling Axe Melee, also don't change range with Octarine Core.
> 
> The 7.32 patch changes Bristle's AOE to "Scepter radius changed from 800 to current Cast Range of Viscous Nasal Goo"

I think the patch notes is a bit poorly worded. Default Goo Cast Range is 650, previously with scepter this increased to 800. Now it is the same as Default 650 cast range with scepter.

**Comment by shaileshgarg** [comment]
This issue can be closed.
